components/automate-ui/src/app/modules/team/team-details/team-details.component.spec.ts

```diff
@@ -151,34 +151,35 @@ describe('TeamDetailsComponent', () => {
     expect(store.dispatch).toHaveBeenCalledWith(new GetProjects());
   });
 
-  it('initializes dropdown with those included on the team checked', () => {
-    spyOn(store, 'dispatch').and.callThrough();
-    const teamProjects = ['b-proj', 'd-proj'];
-    const team: Team = { id: targetId, guid: 'any', name: 'any', projects: teamProjects };
-    store.dispatch(new GetTeamSuccess(team));
-    expect(store.dispatch).toHaveBeenCalledWith(new GetProjects());
+  using([
+    ['no projects', []],
+    ['one project', ['proj-one']],
+    ['multiple projects', ['p1', 'p2', 'p3', 'p4']]
+  ], function (description: string, projects: string[]) {
+    it(`initializes dropdown with those included on the team for ${description}`, () => {
+      const team: Team = { id: targetId, guid: 'any', name: 'any', projects };
+      store.dispatch(new GetTeamSuccess(team));
+
+      expect(component.team.projects).toEqual(projects);
```

Comment:
The contents of `team.projects` is the input to the ProjectDropdownComponent, so we win if that matches the expectation.